pkgs/tools/backup/borg/default.nix

```diff
@@ -44,11 +44,20 @@ python3Packages.buildPythonApplication rec {
     make -C docs man
     mkdir -p $out/share/man
     cp -R docs/_build/man $out/share/man/man1
+
+    mkdir -p $out/share/bash-completion/completions
+    cp scripts/shell_completions/bash/borg $out/share/bash-completion/completions/
+
+    mkdir -p $out/share/fish/completions
+    cp scripts/shell_completions/fish/borg.fish $out/share/fish/completions/
```

Comment:
I think this should be `$out/share/fish/vendor_completions.d`, same below.

Reply:
Done.